src/enums/variant-payloads.md

```diff
@@ -9,8 +9,15 @@ You can define richer enums where the variants carry data. You can then use the
 
 <details>
 
+Key Points:
+  
+* This might be the first time seeing pattern matching so perhaps introduce the topic, more details are provided in further slides.
```

Comment:
Please take this out this line: let the notes _add_ information to the reader (who can be both a student and an instructor). Avoid phrases like "This might" and "so perhaps".